storage/response_cache.go

```diff
@@ -101,9 +99,6 @@ func (rc *ResponseCache) GetResponse(e engine.Engine, cmdID proto.ClientCmdID, r
 // return an error. The copy is done directly using the engine instead
```

Comment:
while you're at it, can you add to the main comment for the response cache that it's not thread safe (because access to it is serialized through raft)?